lib/tool-bar-view.coffee

```diff
@@ -46,29 +44,38 @@ module.exports = class ToolBarView extends View
       @updatePosition 'Left'
       atom.config.set 'tool-bar.position', 'Left'
 
-    atom.config.observe 'tool-bar.iconSize', (newValue) =>
+    @subscriptions.add atom.config.observe 'tool-bar.iconSize', (newValue) =>
       @updateSize newValue
 
-    atom.config.onDidChange 'tool-bar.position', ({newValue, oldValue}) =>
+    @subscriptions.add atom.config.onDidChange 'tool-bar.position', ({newValue, oldValue}) =>
       @show() if atom.config.get 'tool-bar.visible'
 
-    atom.config.onDidChange 'tool-bar.visible', ({newValue, oldValue}) =>
+    @subscriptions.add atom.config.onDidChange 'tool-bar.visible', ({newValue, oldValue}) =>
       if newValue then @show() else @hide()
 
     if atom.config.get 'tool-bar.visible'
       @show()
 
-    @.element.addEventListener 'scroll', @drawGutter
+    @on 'scroll', @drawGutter
```

Comment:
[`remove`](https://api.jquery.com/remove/) not only removes the element from the DOM, but also removes all jQuery-added event handlers. Since, I use `remove()` in `destroy()`, we get event handler removal for free.

---

I'd be more than happy to put together another PR that removes `space-pen` all together.